### ISSUE TITLE
copy_local behaviour

### ISSUE THREAD

**Issue by WebDucer** [fsprojects/Paket#972]
In the `paket.dependencies` I have the following declaration:

```
source https://www.nuget.org/api/v2/

copy_local: false

nuget MvvmLightLibs
nuget Newtonsoft.Json
nuget NLog.Config
nuget NLog
...
nuget FluentAssertions copy_local: true
nuget NUnit copy_local: true
nuget Moq copy_local: true
...
```

I define global that I don't need the local copy of the libraries. I mark `copy_local:true` in the `paket.references` of the project that has the direct dependency on the library. this work very well.

All test projects need always some dependencies (NUnit, Moq etc.). I thoght if I mark them `copy_local:true` in the `paket.depencencies` this should work. But it doesn't. The references in this projects are not marked for copy local. Is this a bug or wanted behaviour?

**Comment by forki** [comment]
that sounds like a bug. will check

**Comment by forki** [comment]
I can reproduce

**Comment by forki** [comment]
fixed in 1.26.1

**Comment by WebDucer** [comment]
Works!
